docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -80,9 +83,9 @@ Sycamore will automatically start processing the new data. The processing job is
 
 `No changes at [datetime] sleeping`
 
-3. Use the demo UI for conversational search. Using your internet browser, visit http://localhost:3000. You can interact with the demo UI while data is being added to the index, but the data won't all be available until the job is done.
+3. Use the demo query UI for conversational search. Using your internet browser, visit: `http://localhost:3000`. You can interact with the demo query UI while data is being added to the index, but the data won't all be available until the job is done. How to use the UI:
 
-* Create a new conversation. Enter the name for your conversation in the text box in the left "Conversations" panel, and hit enter or click the "add convo" icon on the right of the text box.
+* Create a new conversation. Enter the name for your conversation in the text box in the left "Conversations" panel, and hit enter or click the "Add convo" icon on the right of the text box. 
```

Comment:
"Add convo" ->  "Add conversation"?

(I don't see a popup that calls the icon the former, so I'd writte it out)

Reply:
Ack